src/scripting/Scripting.cpp

```diff
@@ -396,7 +396,18 @@ void Scripting::Initialize()
         Overlay::Get().Log(oss.str());
     };
 
-    m_lua.do_file((Options::Get().Path / "scripts" / "autoexec.lua").string());
+
+    // execute autoexec.lua inside our default script directory
+    std::filesystem::path defPath = Options::Get().Path / "scripts";
```

Comment:
This is the same path as ScriptsPath, I don't understand what this is supposed to accomplish?